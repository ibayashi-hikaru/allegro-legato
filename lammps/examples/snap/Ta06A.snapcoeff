../../potentials/Ta06A.snapcoeff