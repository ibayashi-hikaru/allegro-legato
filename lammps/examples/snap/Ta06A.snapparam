../../potentials/Ta06A.snapparam