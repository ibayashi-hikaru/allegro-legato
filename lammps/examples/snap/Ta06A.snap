../../potentials/Ta06A.snap